src/test/kotlin/com/zopa/ktor/opentracing/KtorOpenTracingTest.kt

```diff
@@ -79,7 +79,7 @@ class KtorOpenTracingTest {
     }
 
     @Test
-    fun `Span is not renamed if param value is not unique in path`() = withTestApplication {
```

Comment:
Can we add a note that that is expected but not ideal behaviour

Reply:
yes , good idea

Reply:
Just pushed https://github.com/zopaUK/ktor-opentracing/pull/25/commits/6c2904fcd19d6b0225d651b12a03af53fff2ed07